indico/modules/events/__init__.py

```diff
@@ -191,6 +191,8 @@ class SubmitterPermission(ManagementPermission):
     name = 'submit'
     friendly_name = _('Submission')
     description = _('Grants access to materials and minutes.')
```

Comment:
Let's clarify this, the current one is misleading since it sounds like you may need it to download these things.

How about this instead? "Grants management rights for event-wide materials and minutes"

If you change it, let's update the one for contributions (`SubmitterPermission` in `modules/events/contributions/__init__.py`) as well even though we currently do not show it anywhere:

"Grants management rights for materials and minutes." (since this would always be shown in the context of editing contribution permissions, it's probably not necessary to mention something like "associated with the contribution" there; that should be obvious)